sql/extra/extract_patient_profile_photo_from_db.sql

```diff
@@ -0,0 +1,24 @@
+DROP PROCEDURE IF EXISTS dump_image;
+DELIMITER //
+  CREATE PROCEDURE dump_image()
+  BEGIN
+
+	DECLARE done INT;
```

Comment:
formatting?

Reply:
no, just indentation of the stored procedure